charts/crossplane-azure-upbound/values.yaml

```
@@ -0,0 +1,70 @@
+global:
+  enabled_azure_upbound: true
+
+deploymentRuntimeConfig:
+  enabled: false
+  metadata:
+    name: "upbound-azure-runtime-config"
+    role_arn: ""
+    annotations: {}
+    labels:
+      app.kubernetes.io/managed-by: Helm
```

```suggestion
    labels: {}
```

The label `app.kubernetes.io/managed-by: Helm` is automatically put by helm cli, you don't want this present when using argocd